.github/workflows/macos-sanity.yaml

```diff
@@ -0,0 +1,48 @@
+name: macOS sanity checks
+on:
+  # Only run on pull requests: not pushes
+  pull_request:
+    branches: [ "master", "release/**" ]
+
+jobs:
+  macos-sanity:
+    runs-on: macos-latest
+    env:
+      GOPATH: ${{ github.workspace }}
+      GO111MODULE: off
+    steps:
+      - uses: actions/setup-go@v2
+        with:
+          go-version: "1.10.x"
+
+      - name: Checkout code
+        uses: actions/checkout@v2
+        with:
+          # NOTE: checkout the code in a fixed location, even for forks, as this
+          # is relevant for go's import system.
+          path: ./src/github.com/snapcore/snapd
+
+      - name: Install squashfs from homebrew
+        run: |
+          brew install squashfs
+
+      - name: Install Go package dependencies
+        run: |
+          ./mkversion.sh
+          ./get-deps.sh
+          # extra dependency on darwin:
+          go get golang.org/x/sys/unix
```

Comment:
Would be great to eventually teach get-deps about it

Reply:
done

Reply:
where did you do this?

Reply:
Sorry for the confusion. It's not done, I replied under the wrong comment.